A blog post about Computo (#1177)
ropensci/roweb3: content/blog/2025-10-20-computo/index.md

---
title: Computo - A Journal for Transparent and Reproducible Research in Statistics and Machine Learning
author:
- Julien Chiquet
- François-David Collin
- Marie-Pierre Etienne
- Aymeric Stamm
- Nelle Varoquaux
slug: computo-journal-reproducible-research-statistics-machine-learning
date: '2025-10-20'
tags:
- reproducible research
- open science
- statistics
- machine learning
- R
- community
description: "[Computo](https://computo-journal.org) is a journal promoting methodological, computational, and algorithmic contributions in statistics and machine learning that provide a better understanding of which models or methods are most appropriate to answer specific scientific questions. Computo leverages modern tools in programming and scientific reporting to support more transparent, interactive, and reproducible research outputs."
socialImg: https://computo-journal.org/assets/img/logo_text_vertical.svg
socialAlt: Computo logo
social: A post about the Computo scientific journal by @computo@mathstodon.xyz!

> # editor: ~

Reviewer: ```suggestion
editor: Steffi LaZerte
```

---

> {{< imgtxt src="https://computo-journal.org/assets/img/logo_text_vertical.svg" alt="Computo logo" link="https://computo-journal.org/site/about.html" >}}

Author: In the browser, it works fine. I would not know how to help debug this further.

> {{< imgtxt src="https://computo-journal.org/assets/img/logo_text_vertical.svg" alt="Computo logo" link="https://computo-journal.org/site/about.html" >}}

Reviewer: Oh glad to hear it works for you, perhaps it's a browser-related thing. At any rate Maëlle pushed a fix (it wasn't your problem 😁) so we're hopefully good to go!

[Computo](https://computo-journal.org) is a journal promoting methodological, computational, and algorithmic contributions in statistics and machine learning that provide a better understanding of which models or methods are most appropriate to answer specific scientific questions. Computo leverages modern tools in programming and scientific reporting to support more transparent, interactive, and reproducible research outputs.
{{</ imgtxt >}}

## Why we created Computo

> The idea of Computo was born out of a need to address several challenges in the current scientific publishing landscape, particularly within statistics and machine learning. While the reproducibility crisis is a significant concern, with an increasing volume of publications straining the review process and potentially impacting quality \[1\], Computo also aims to foster a deeper scientific understanding.

Reviewer: ```suggestion
The idea of Computo was born out of a need to address several challenges in the current scientific publishing landscape, particularly within statistics and machine learning. While the reproducibility crisis is a significant concern, with an increasing volume of publications straining the review process and potentially impacting quality [^1], Computo also aims to foster a deeper scientific understanding.
```

> The idea of Computo was born out of a need to address several challenges in the current scientific publishing landscape, particularly within statistics and machine learning. While the reproducibility crisis is a significant concern, with an increasing volume of publications straining the review process and potentially impacting quality \[1\], Computo also aims to foster a deeper scientific understanding.

Author: I will check with the editorial board today and get back to you on this later in the day.

Computo was created to promote methodological and algorithmic contributions that not only are reproducible but also provide a better understanding of which models or methods are most appropriate to answer specific scientific questions. We believe that building up solid scientific knowledge requires both the reproducibility of published results and a clear articulation of their scientific implications. While many journals now require code and data, these are often insufficient on their own to ensure true reproducibility or to fully grasp the scientific contribution.

To address these issues, Computo goes beyond the traditional format of a static (PDF) publication, by placing the reproducibility of numerical results and the clarity of scientific contributions at the center of the publication process.&#x20;

> \[1\] Hanson, M. A., Barreiro, P. G., Crosetto, P., & Brockington, D. (2024). The strain on scientific publishing. Quantitative Science Studies, 5(4), 823-843.

Reviewer: ```suggestion
[^1]: Hanson, M. A., Barreiro, P. G., Crosetto, P., & Brockington, D. (2024). The strain on scientific publishing. Quantitative Science Studies, 5(4), 823-843.
```

Looking ahead, we aim to further enhance the dissemination of scientific knowledge by moving towards greater interactivity, richer data visualizations, and exploring innovative communication models, drawing inspiration from initiatives like the former [distill.pub](https://distill.pub). Our approach is to progressively engage all stakeholders: editors, engineers, authors, and the wider scientific community in this evolution.

Our concrete next steps focus on:

*   **Streamlining Workflows and Tools:** Consolidating our editorial process, potentially within a more community-focused environment, and expanding authoring tools with user-friendly computational environments for pre-submission.
*   **Ensuring Long-Term Access and Innovation:** Deploying a robust platform for long-term reproducibility, including on-the-fly container creation and archiving, while continuing to explore and integrate novel ways to present and interact with scientific findings.

> We look forward to continuing our journey with the support of our community and partners like rOpenSci.

Reviewer: ```suggestion
We look forward to continuing our journey with the support of our community and like-minded organizations like rOpenSci.
```
